src/PresentationalComponents/Cards/OverviewDashbarCard.js

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import {
+  Card,
+  CardBody,
+  Flex,
+  FlexItem,
+  Tooltip,
+  Title,
+  Text,
+  TextVariants,
+} from '@patternfly/react-core';
+import { RouteIcon } from '@patternfly/react-icons/dist/esm/icons/route-icon';
+
+import { InsightsLabel } from '@redhat-cloud-services/frontend-components/InsightsLabel';
+import { QuestionTooltip } from '../Common/Common';
+import RuleLabels from '../Labels/RuleLabels';
+import { strong } from '../../Utilities/intlHelper';
+import messages from '../../Messages';
+import {
+  PATHWAYS,
+  INCIDENTS,
+  IMPORTANT_RECOMMENDATIONS,
+  CRITICAL_RECOMMENDATIONS,
+  INCIDENT_TAG,
+  IMPORTANT_TAG,
+  CRITICAL_TAG,
+  TOTAL_RISK_LABEL_LOWER,
+  SEVERITY_MAP,
+} from '../../AppConstants';
+
+/*****************************************/
+// this component returns an "Important" or "Critical" label with an appropriate formatted tooltip message
+// It receives a typeOfTag prop, which is a number, and uses it to get the appropriate label and corresponding tooltip message
+const GetTagLabelWithTooltip = ({ typeOfTag, formatMessage }) => (
```

Comment:
Do also try to keep one component per file. This could be for example in `PresentationalComponents/GetTagLabelWithTooltip/GetTagLabelWithTooltip.js` (or better `PresentationalComponents/TagLabelWithTooltip/TagLabelWithTooltip.js`)

Reply:
Will do 👍🏼